### PR TITLE
Sync with Fybrik recent changes

### LINKED ISSUE #8: Support helm 3.7
In helm release 3.7 the commands related to OCI registry support have changed. More information about the change can be found in https://github.com/helm/community/blob/main/hips/hip-0006.md.
This issue is about supporting the new commands related to OCI registry in helm 3.7.

### PR DESCRIPTION
**Pull request by revit13**
Update resources.
Update to helm 3.7
Update README.md file.

Closes #8

### CODE REVIEW

File: fybrikapplication.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ spec:
    workloadSelector:
      matchLabels: {}
   appInfo:
-    intent: fraud-detection
+    intent: Fraud Detection
```
Comment (roytman) [inline fybrikapplication.yaml]: you should quotes strings with spaces.

File: hack/make-rules/helm.mk [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	$(ABSTOOLBIN)/helm uninstall ${HELM_RELEASE} || true
+
+.PHONY: helm-install
+helm-install: $(TOOLBIN)/helm
+	$(ABSTOOLBIN)/helm install ${HELM_RELEASE} ${CHART_LOCAL_PATH} ${HELM_VALUES}
+
+
+# example for helm chart push:
+# helm package fybrik-template -d /tmp/ --version 0.7.0
+# helm push /tmp/fybrik-template-0.7.0.tgz oci://localhost:5000/fybrik-system/
```
Comment (roytman) [inline hack/make-rules/helm.mk:50]: why the version is 0.7.0 ?

File: fybrikapplication.yaml (hunk 1 above)
Comment (revit13) [reply fybrikapplication.yaml]: OK btw I took it from https://fybrik.io/v0.5/samples/notebook/ - please see `Create a FybrikApplication resource for the notebook` section

File: hack/make-rules/helm.mk (hunk 2 above)
Comment (revit13) [reply hack/make-rules/helm.mk:50]: This is just an example...

File: fybrikapplication.yaml (hunk 1 above)
Comment (roytman) [reply fybrikapplication.yaml]: it has `intent: fraud-detection` as it was here, without spaces

File: Makefile [hunk 3]
```diff
@@ -5,15 +5,13 @@ export DOCKER_PASSWORD ?=
 export DOCKER_HOSTNAME ?= ghcr.io
 export DOCKER_NAMESPACE ?= fybrik
 export DOCKER_TAGNAME ?= 0.0.0
+export DOCKER_IMG_NAME := hello-world-module
```
Comment (shlomitk1) [inline Makefile]: Why do we need this? DOCKER_NAME should be enough.

File: Makefile [hunk 4]
```diff
@@ -5,15 +5,13 @@ export DOCKER_PASSWORD ?=
...
 
-DOCKER_IMG_NAME ?= hello-world-module
-DOCKER_CHART_IMG_NAME ?= hello-world-module-chart
 DOCKER_FILE ?= Dockerfile
 DOCKER_CONTEXT ?= .
-HELM_TAG ?= ${DOCKER_TAGNAME}
+DOCKER_NAME ?= hello-world-module
 
 APP_IMG ?= ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/${DOCKER_IMG_NAME}:${DOCKER_TAGNAME}
```
Comment (shlomitk1) [inline Makefile]: Use DOCKER_NAME instead

File: Makefile [hunk 5]
```diff
@@ -5,15 +5,13 @@ export DOCKER_PASSWORD ?=
...
-CHART_IMG ?= ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/${DOCKER_CHART_IMG_NAME}:${HELM_TAG}
```
Comment (shlomitk1) [inline Makefile:16]: How is the chart pushed to the registry?

File: README.md [hunk 6]
```diff
@@ -10,14 +10,14 @@ The configuration for the chart is in the values file.
...
-- Helm 3.0.0+
+- Helm 3.7.x
 
 ## Installation
 
 ### Modify values in Makefile
 
 In `Makefile`:
-- Change `DOCKER_USERNAME`, `DOCKER_PASSWORD`, `DOCKER_HOSTNAME`, `DOCKER_NAMESPACE`, `DOCKER_TAGNAME`, `DOCKER_IMG_NAME`, and `DOCKER_CHART_IMG_NAME` to your own preferences.
+- Change `DOCKER_USERNAME`, `DOCKER_PASSWORD`, `DOCKER_HOSTNAME`, `DOCKER_NAMESPACE`, `DOCKER_TAGNAME`, `DOCKER_NAME`, `HELM_TAG` to your own preferences.
```
Comment (shlomitk1) [inline README.md:20]: Where is HELM_TAG used?

File: fybrikapplication.yaml (hunk 1 above)
Comment (revit13) [reply fybrikapplication.yaml]: @roytman sorry I was referring to the dev version https://fybrik.io/dev/samples/notebook/

File: Makefile (hunk 3 above)
Comment (revit13) [reply Makefile]: Current, will remove it. Thanks

File: fybrikapplication.yaml (hunk 1 above)
Comment (roytman) [reply fybrikapplication.yaml]: I'd quote both cases, including the one in the docs

**Review by roytman — APPROVED** [review]

**Pull request merged**